src/js/components/awardv2/contract/AdditionalInfo.jsx

```diff
@@ -27,13 +27,23 @@ export default class AdditionalInfo extends React.Component {
     }
     render() {
         const awardData = this.props.overview;
+        const periodOfPerformanceData = awardData.periodOfPerformance || awardData.dates;
+        const parentAwards = awardData.parentAwardDetails;
         const data = {
+            agencyDetails: {
+                'Awarding Department': `${awardData.awardingAgency.toptierName} (${awardData.awardingAgency.toptierAbbr})`,
```

Comment:
You can use the formattedToptier property here to add the acronym and paretheses.